app/models/routable_artefact.rb

```diff
@@ -0,0 +1,87 @@
+require 'plek'
+require 'gds_api/router'
+
+class RoutableArtefact
```

Comment:
That's right - it's straight from Panopticon, and we're expecting to remove it at a later date.